bot/utils/scheduling.py

```diff
@@ -157,18 +157,20 @@ def _task_done_callback(self, task_id: t.Hashable, done_task: asyncio.Task) -> N
                 self._log.error(f"Error in task #{task_id} {id(done_task)}!", exc_info=exception)
 
 
-def create_task(*args, **kwargs) -> asyncio.Task:
+def create_task(suppress_exceptions: t.Tuple[Exception], *args, **kwargs) -> asyncio.Task:
     """Wrapper for `asyncio.create_task` which logs exceptions raised in the task."""
     task = asyncio.create_task(*args, **kwargs)
-    task.add_done_callback(_log_task_exception)
+    task.add_done_callback(partial(_log_task_exception, suppress_exceptions=suppress_exceptions))
     return task
 
 
-def _log_task_exception(task: asyncio.Task) -> None:
+def _log_task_exception(task: asyncio.Task, suppress_exceptions: t.Tuple[Exception] = ()) -> None:
```

Comment:
`suppressed_exceptions` would be a better name since the current name sounds more like a Boolean.